lesson28/script.js

```diff
@@ -0,0 +1,38 @@
+//1.Реализовать класс, описывающий окружность. В классе должны быть следующие компоненты:
+
+// поле, хранящее радиус окружности;
+// get-свойство, возвращающее радиус окружности;
+// set-свойство, устанавливающее радиус окружности;
+// get-свойство, возвращающее диаметр окружности;
+// метод, вычисляющий площадь окружности;
+// метод, вычисляющий длину окружности.
+
+class SomeClass {
```

Comment:
клас, описывающий окружность, можно и назвать Окружность =)
```suggestion
class Circle {
```